src/boardgameprojectfile/boardgameprojectfile.cpp

```diff
@@ -0,0 +1,55 @@
+#include "boardgameprojectfile.h"
+#include <QFile>
+#include <QDataStream>
+#include <QString>
+#include <QDebug>

```

Comment:
Debug stuff should be deleted

Reply:
It is used to display an error while saving the file